doc/nrf/libraries/others/event_manager.rst

```diff
@@ -30,9 +30,6 @@ As part of this communication, listeners can process events differently based on
 The Event Manager provides API for defining, creating, and subscribing events.
 See `Implementing an event type`_ for details about how to create custom event types.
 
-.. note::
```

Comment:
The same note must be removed from `profiler.rst` (there used to be the same limitation). Sorry that I did not notice it in previous reviews.